docs/FilePropertyManagement.md

```diff
@@ -0,0 +1,150 @@
+  2. A group definition, to overriding the default by specifying using DBB's file property syntax using a pattern filter; or through a mapping to a language definition file to override the defaults, like [application-conf/languageDefinitionMapping.properties](../samples/MortgageApplication/application-conf/languageDefinitionMapping.properties). 
+  3. An individual file-level definition to override a single parameter leveraging Dependency Based Builds file properties syntax using [application-conf/file.properties](../samples/application-conf/file.properties) or multiple parameters for a specific file using the individual property file, like [epsmlist.cbl.properties](../samples/MortgageApplication/properties/epsmlist.cbl.properties).
+
+zAppBuild comes with various strategies that can be combined via an order of precedence. The following table summarizes the strategies for overriding file properties from highest to lowest precedence:
+
+||Strategy|Use case|
+|-|-|-|
+|1.|Individual file properties|Override one or multiple build parameters for individual files|
+|2.|Language definition mapping|Override and define one or multiple build parameters for a group of mapped application artifacts|
+|3.|DBB file properties|Override a single build parameter for individual files or a grouped list of application artifacts|
+|4.|Default properties|General build properties used when no overrides are defined|
+
+To understand the order of precedence, think of this as a merge of the property configurations. For example, if both individual file properties and a language definition mapping are configured for a file, then the properties defined through the individual file property definition take precedence, but are also merged with other properties defined by the language definition mapping and the default properties.
+
+
+
+## Default properties
+
+Default properties can be set in the corresponding language properties file.  For example, the COBOL file properties can be set in [application-conf/Cobol.properties](../samples/application-conf/Cobol.properties), while the Assembler file properties can be set in [application-conf/Assembler.properties](../samples/application-conf/Assembler.properties), and so on.
+
+zAppBuild is currently proposing to store these properties within the application repository and let the application team have control over these files. If you are looking for a more centralized way to manage the default options for all applications, you can move these definitions into the zAppBuild build framework itself by either merging them into the appropriate language property file under [build-conf](../build-conf/) or store them in a separate directory within zAppBuild itself and leverage the `applicationConfRootDir` property in [build-conf/build.propertiees](../build-conf/build.properties).
```

Comment:
Should we say something about the order of precedence between `application-conf` vs `build-conf` here?

Reply:
Yes, we can say, that the default order of loading the properties files is `build-conf` first and `application-conf` second.

Reply:
Just to confirm: This means that, by default, properties defined in `application-conf` would override those defined in `build-conf` - is that correct?

Reply:
Confirmed with @dennis-behm